test/VisionModel/vgg.jl

```diff
@@ -0,0 +1,11 @@
+sing Flux
+using FluxModels
+using Test
+
+@testset "ResNet" begin
+  for model in [vgg11, vgg11bn, vgg13, vgg13bn, vgg16, vgg16bn, vgg19, vgg19bn]
+    m = model()
+    @test size(m(rand(Float32, 256, 256, 3, 50))) == (1000, 50)
+  end
+end
```

Comment:
Can you change this to "VGG", and remove the `begin ... end` block. You can directly go into a for-loop like:
```julia
@testset "VGG" for model in [...]
    # tests
end
```

Going directly into a loop means that if a single iterations of the loop fails, then the test output report will recognize that and format the report accordingly.